### PR TITLE
[cli] Remove initial iteration of `vc build` and Plugins

### PR DESCRIPTION
**Pull request by TooTallNate**
The `vercel build` command is being restructured, so we'll remove it for now so that this initial iteration isn't included as part of the stable CLI release.

### CODE REVIEW

File: packages/cli/src/commands/init/init.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ type Example = {
 
 const EXAMPLE_API = 'https://now-example-files.zeit.sh';
 
+export const isBeta = true;
```
Comment (styfle) [inline packages/cli/src/commands/init/init.ts]: Why is `vercel init` beta?

Comment (TooTallNate) [reply packages/cli/src/commands/init/init.ts]: Whoops, didn't mean to commit that.

```suggestion
```

**Review by styfle — APPROVED** [review]

**Review by javivelasco — APPROVED** [review]

**Review by gdborton — APPROVED** [review]

**Review by styfle — COMMENTED** [review]
Lets also remove the `.output` change to `.gitignore`

**Review by styfle — APPROVED** [review]

**Review by EndangeredMassa — APPROVED** [review]
There are a couple of references to `vercel build` in comments, but I don't think it's blocking.

- packages/cli/src/util/projects/project-settings.ts
- packages/build-utils/src/detect-file-system-api.ts

**Pull request merged**